scripts/ai/AIDriveStrategyCombineCourse.lua

```diff
@@ -1331,7 +1332,10 @@ function AIDriveStrategyCombineCourse:startTurn(ix)
 
     -- Combines drive special headland corner maneuvers, except potato and sugarbeet harvesters
     if self.turnContext:isHeadlandCorner() then
-        if self.combineController:isPotatoOrSugarBeetHarvester() then
+        if self.combineController:isTowed() then
+            self:debug('Headland turn but this is a towed harvester using normal turn maneuvers.')
+            AIDriveStrategyCombineCourse.superClass().startTurn(self, ix)
+        elseif self.combineController:isPotatoOrSugarBeetHarvester() then

```

Comment:
I guess we still need a replacement for `:isPotatoOrSugarBeetHarvester()`, as this function doesn't work for example for the same harvester, but as mods with different fill types.
Maybe a TODO comment, so we don't forget this?